mautic/mautic
# TPROD-321 - NotificationBundle - Complete PHP Templates to Twig

This contains the existing PRs for the NotificationBundle along with all the final updates and changes to complete the upgrade of the NotificationBundle



| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create or Edit a Campaign
3. Open the Campaign Builder
4. Verify that the action **Send mobile notification** can be added to the campaign
5. Verify that the action **Send web notification** can be added to the campaign
